BR Simple/DetailsViewController.swift

```diff
@@ -6,12 +6,56 @@
 //
 
 import UIKit
+import MapKit
 
-final class DetailsViewController: UIViewController {
+final class DetailsViewController: UIViewController, MKMapViewDelegate {
+    @IBOutlet var image: UIImageView!
+    @IBOutlet var name: UILabel!
+    @IBOutlet var category: UILabel!
+    @IBOutlet var address: UILabel!
+    @IBOutlet var phone: UILabel!
+    @IBOutlet var social: UILabel!
+    @IBOutlet var mapKitView: MKMapView!
+    
+    var restaurant: RestaurantsModel?
+    
     
     override func viewDidLoad() {
         super.viewDidLoad()
+        updateUI()
+        updateMap()
+        
+        view.backgroundColor = .lightGray
+    }
+    
+    func updateUI() {
+        
+        self.name.text = restaurant?.restaurantsName
+        self.category.text = restaurant?.restaurantsCategory
+        self.address.text = restaurant?.location.address
+        self.phone.text = restaurant?.contact?.formattedPhone
+        self.social.text = restaurant?.contact?.twitter
+    }
+    
+    func updateMap() {
+        
+        let lattitude: CLLocationDegrees = (restaurant?.location.lat)!
+        let longitude: CLLocationDegrees = (restaurant?.location.lng)!
```

Comment:
use guard let to safely unwrap 'restaurant', if it is nil, get out from the function